### PR TITLE
Disable sync correction by default for SyncPlay

### LINKED ISSUE #4972: Disable sync correction for SyncPlay by default
**Describe The Change**

The sync correction functionality seems to cause a lot of issues when most people probably don't care about super precise syncing. We already disabled this by default for mobile #2946.

### PR DESCRIPTION
**Pull request by DinuD**
**Changes**

Sets the default value for sync correction to false for everything.
**Issues**

Fixes #4972

### CODE REVIEW

**Comment by DinuD** [comment]
This is my first open-source contribution so I'm not entirely sure if I should ask for someone to take a look at this or just wait until it gets merged, or if this will get merged before a certain release. Thanks and apologies if I've done something wrong!

**Comment by thornbill** [comment]
You are fine @DinuD. I am just behind on reviews. I should get to this at some point in the next week or two. 😅

**Review by thornbill — APPROVED** [review]

**Pull request merged**
